flipperdevices/flipperzero-firmware
# Add GPIO control through RPC

# What's new

- Add GPIO control through the RPC interface

# Verification 

- Use the protobuf definition in https://github.com/flipperdevices/flipperzero-protobuf/pull/30
- You can also use the bindings from https://github.com/flipperdevices/flipperzero_protobuf_py/pull/5
- Ensure the behavior is adequate

# Checklist (For Reviewer)
